### DISCUSSION TITLE
Allow Blast Furnace to turn tools back into ingots

### DISCUSSION THREAD

**Discussion by dwentz89** [TechReborn/TechReborn#2851] [Show and tell]
The Industrial Blast Furnace can already melt armor back into ingots, so I think it should be able to do the same with held items like pickaxes and swords. And maybe these could give an extra dark ash since the stick is getting burnt up.

**Comment by Ayutac** [comment]
In principal yes, but currently we are not sure how to keep the return on armor (see #2678), so I would put this suggestion on hold until then.

**Comment by Ayutac** [comment]
by now also see #2893

**Reply by Ayutac** [reply]
has been implemented by now
